### DISCUSSION TITLE
Default to "edit" title in new item screen?

### DISCUSSION THREAD

**Discussion by canadaduane** [ckipp01/skan#37] [Feature Request] [answered]
I wonder if it would make sense to put the focus on the Title input so you can start typing right away after pressing "n" for new item.

**Comment by ckipp01** [comment]
Ahh, this is a great idea. I think I'll do this.

**Comment by ckipp01** [comment, marked as answer]
Implemented in https://github.com/ckipp01/skan/pull/38.
